examples/window.rs

```diff
@@ -983,6 +1063,14 @@ const KEY_BINDINGS: &[Binding<&'static str>] = &[
     // C.
     Binding::new("C", ModifiersState::CONTROL, Action::NextCursor),
     Binding::new("C", ModifiersState::ALT, Action::NextCustomCursor),
+    #[cfg(web_platform)]
+    Binding::new("C", ModifiersState::SHIFT, Action::UrlCustomCursor),
```

Comment:
We generally don't use `SHIFT` as you can see, so you can type more or less normally.

Reply:
Any alternative suggestion?
I didn't want to move away from the "C" key.

Reply:
`ALT | SHIFT` should be fine, just don't use plain `SHIFT`

Reply:
Any alternative suggestion to the plain `SHIFT`?

Reply:
`CONTROL` is taken.
`SUPER` is probably not something we want here, browsers will react funny.
I took `ALT`.